sim/dex.ts

```diff
@@ -471,6 +479,35 @@ export class ModdedDex {
 		return learnsetData;
 	}
 
+	getDescs(table: keyof TextTableData, id: ID, dataEntry: AnyObject) {
```

Comment:
These APIs should really be `async`

Reply:
It's temporary. monkfish didn't want to change the API yet and this turned out to be reasonably straightforward.